ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -1015,7 +1018,7 @@ function seriesRenderCanvas(this: Series): void {
      * @private
      */
     function processPoint(
-        d: (number|Array<number>|Record<string, number>),
+        d: (number|Array<number>|Record<string, number>|number),
```

Comment:
Istn't the `number` type already covered in the beginning of the line?

Reply:
Yes, good point!